app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format.")
+        return
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file != destination_file:
+        with open(source_file, "r") as file_in, \
```

Comment:
please, avoid using \, better use brackets:
..."Accessible", (
"the page is...
)